File: DemoClient/projects/ng-realtime-database/src/lib/models/command/execute-command.ts
import {CommandBase} from './command-base';

export class ExecuteCommand extends CommandBase {
  actionHandlerName: string;
  actionName: string;
  parameters: any[];

  constructor(actionHandlerName: string, actionName: string, parameters: any[]) {
    super('ExecuteCommand');
    this.actionHandlerName = actionHandlerName;
    this.actionName = actionName;
    this.parameters = parameters;
  }
}

File: SapphireDb/Command/Execute/ExecuteCommand.cs
using Newtonsoft.Json.Linq;

namespace SapphireDb.Command.Execute
{
    public class ExecuteCommand : CommandBase
    {
        public string ActionHandlerName { get; set; }

        public string ActionName { get; set; }

        public JToken[] Parameters { get; set; } = new JToken[0];
    }
}
